ci/travis/setup.sh

```diff
@@ -33,3 +33,15 @@ if [ ! -f "$HOME/.cargo/bin/kcov" ]; then
     install src/kcov $HOME/.cargo/bin/kcov
     cd $TRAVIS_BUILD_DIR
 fi
+
+
+### Setup PyPy ################################################################
+
+wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
```

Comment:
Maybe we should use `if [[ $FEATURES == *"pypy"* ]]` here?

Reply:
👍